trulens_eval/trulens_eval/app.py

```diff
@@ -588,19 +607,19 @@ def _get_method_path(self, obj: object, func: Callable) -> JSONPath:
         funcs = self.instrumented_methods.get(id(obj))
 
         if funcs is None:
-            logger.debug(
+            logger.warn(
                 f"A new object of type {type(obj)} at 0x{id(obj):x} is calling an instrumented method {func}. "
                 "The path of this call may be incorrect."
             )
             try:
                 _id, f, path = next(iter(self._get_methods_for_func(func)))
             except Exception:
-                logger.debug(
+                logger.warn(
                     "No other objects use this function so cannot guess path."
```

Comment:
Is guessing the best language to use here?

Reply:
I think so, this point of the code is meant to guess a path (pick one without being sure it is correct).